grpc/requests_common.hpp

```diff
@@ -0,0 +1,68 @@
+#pragma once
+
+#include <thread>
+#include <utility>
+#include <memory>
+
+#include <blackhole/attribute.hpp>
+
+#include <grpcpp/grpcpp.h>
+#include <grpcpp/server.h>
+#include <grpcpp/server_builder.h>
+#include <grpcpp/server_context.h>
+
+#include "elliptics.grpc.fb.h"
+
+#include "elliptics/utils.hpp"
+
+#include "library/elliptics.h"
+#include "library/logger.hpp"
+#include "library/n2_protocol.hpp"
+
+namespace fb = flatbuffers;
+namespace ell = ioremap::elliptics;
+
+namespace grpc_dnet {
+
+struct service_provider_info {
```

Comment:
Why is it needed?

Reply:
Not to copy stable information from call to call: let's all call structs refer to some "global" data.